src/consensus/pbft_manager.hpp

```diff
@@ -238,7 +238,8 @@ class PbftManager : public std::enable_shared_from_this<PbftManager> {
   std::condition_variable stop_cv_;
   std::mutex stop_mtx_;
 
-  std::queue<std::pair<SyncBlock, dev::p2p::NodeID>> sync_queue_;
+  std::deque<std::pair<SyncBlock, dev::p2p::NodeID>> sync_queue_;
+  std::atomic<uint64_t> sync_period_{0};
```

Comment:
The last synced period is sync_queue_.back().first.pbft_blk->getPeriod(), don't need extra sync_period_